code/game/objects/items/stacks/rods.dm

```diff
@@ -11,7 +15,7 @@
 	throw_speed = 3
 	throw_range = 7
 	materials = list(MAT_METAL=1000)
-	max_amount = 60
+	max_amount = 50

```

Comment:
I wanted to standardize it as the ores, glass, plasteel and metal all have max 50. Updated changelog.